templates/todo/task_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Task</h1>
```

Comment:
The template tag {{ object|yesno:"Update,Create" }} you're attempting to use does not exist in Django. Instead, you can achieve the same functionality using conditional logic directly in the template like this:
```
{% if object %}
    Update Task
{% else %}
    Create Task
{% endif %}
```